mobile/src/main.jsx

```diff
@@ -37,7 +37,7 @@ const renderAppWithPersistedState = persistedState => {
           cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
           initBar()
         } else {
-          onError(store.dispatch, store.getState)()
+          onError(store.dispatch, store.getState)({ message: 'Client has been revoked' })
```

Comment:
I write that code this way as it brings to light the need of a refactoring. It is a big smell and I would like to fix the bug, not to refactor the whole error management in pouchDB replication.

This is not an easy move as it involves a lot of different actors:
- the different cozy-stack's error ;
- the cozy-client-js layer ;
- the pouchDB internal layer ;
- the cozy-files-v3 handling those all together.

It worth to be refactored once and for all now we have all the actors in the place, imho.